### PR TITLE
Closes #123 Add support for AISMessage27

### LINKED ISSUE #123: Support for the AISMessage27
Hello @ktuukkan  :)

Do you plan to support for AISMessage27 https://www.navcen.uscg.gov/?pageName=AISMessage27 in the near future?
If not, any particular tips if I would like to jump on it and try to add this support?

Best regards
Krzysiek B.

### PR DESCRIPTION
**Pull request by liosedhel**
Hello :) 

I added the support for AIS Message 27. Issue reported in #123 
@ktuukkan any chance to look at this? (and possibly release new lib version soon?)

Best regards,
Krzysztof

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -145,6 +145,7 @@ along with Java Marine API. If not, see <http://www.gnu.org/licenses />.
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <version>3.0.1</version>
                 <configuration>
+                    <source>8</source>
```
Comment (ktuukkan) [inline pom.xml:148]: Duplicate source tag?

**Review by ktuukkan — APPROVED** [review]
Looking good, thanks!

File: pom.xml (hunk 1 above)
Comment (liosedhel) [reply pom.xml]: Ups, removed, thanks :)

**Comment by liosedhel** [comment]
Fixed the comment. @ktuukkan thanks for taking a look :) Ready to merge? When do you plan a new release?

**Comment by ktuukkan** [comment]
Thanks, I'll merge now and build a new snapshot for you to use before the next release. I'll check and align all the recent changes before making it, might take a while as I've also considered migrating the releases from Sonatype to Github first, but that could also wait depending on how much I have time.

**Pull request merged**